### PR TITLE
Describe web Gantt 

### PR DESCRIPTION
**Pull request by AnnaKondratova**

### CODE REVIEW

File: interface-elements-for-web/articles/gantt/data-editing/modify-dependencies.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+title: Add and Delete Dependencies
+author: Anna Kondratova
+---
+# Add and Delete Dependencies
+
+## Add Dependency
+
+A dependency type specifies which edges (right or left) of the preceding and succeeding tasks serve as the start and end points for the dependency.
```
Comment (albertov05) [inline interface-elements-for-web/articles/gantt/data-editing/modify-dependencies.md:9]:
```suggestion
A dependency type specifies which edges (right or left - for preceding and succeeding tasks) serve as the start and end points for the dependency. 
```

File: interface-elements-for-web/articles/gantt/data-editing/modify-dependencies.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+title: Add and Delete Dependencies
+author: Anna Kondratova
+---
+# Add and Delete Dependencies
+
+## Add Dependency
+
+A dependency type specifies which edges (right or left) of the preceding and succeeding tasks serve as the start and end points for the dependency. 
+
+Hold on the preceding task's required edge (marked with a circle) and drag the line to the required edge of the succeeding task.
```
Comment (albertov05) [inline interface-elements-for-web/articles/gantt/data-editing/modify-dependencies.md:11]: By "drag the line" do you mean "drag the connector"? If so, could we say:

Move the mouse pointer above the preceding task's edge (marked with a circle) and drag the connector to the edge of the next task.

File: interface-elements-for-web/articles/gantt/data-editing/modify-resources.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+Double-click a task or use the [context menu](~/interface-elements-for-web/articles/gantt/context-menu.md) to open the 'Task Details' dialog window to manage resources.
+
+## List of Available Resources
+
+Click the "Resource Manager" button to modify the list of available resources.
+
+![](~/interface-elements-for-web/images/Gantt/resource-manager.png)
+
+Type the name of a new resource and click **Add** to add a new resource to the list.
```
Comment (albertov05) [inline interface-elements-for-web/articles/gantt/data-editing/modify-resources.md:15]:
```suggestion
Type the name of a new resource and click **Add** to add it to the list.
```

File: interface-elements-for-web/articles/gantt/data-editing/modify-resources.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+![](~/interface-elements-for-web/images/Gantt/new-resource.png)
+
+Select a resource and click **Delete** to delete the resource from the list.
+
+![](~/interface-elements-for-web/images/Gantt/delete-resource.png)
+
+## Link Resource to Task
+
+Click the "Resources" field to open the drop-down list with available resources.
```
Comment (albertov05) [inline interface-elements-for-web/articles/gantt/data-editing/modify-resources.md:25]:
```suggestion
Click the "Resources" field to open a drop-down list with available resources.
```

File: interface-elements-for-web/articles/gantt/data-editing/modify-resources.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+![](~/interface-elements-for-web/images/Gantt/resource-list.png)
+
+## Remove Resource From Task
```
Comment (albertov05) [inline interface-elements-for-web/articles/gantt/data-editing/modify-resources.md:29]:
```suggestion
## Remove Resource from Task
```

File: interface-elements-for-web/articles/gantt/data-editing/modify-resources.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+Click the cross icon on the right side of the resource to remove it from the task.
```
Comment (albertov05) [inline interface-elements-for-web/articles/gantt/data-editing/modify-resources.md:31]:
```suggestion
Click the 'cross' icon (on the right side of the resource) to remove it from the task. 
```

File: interface-elements-for-web/articles/gantt/data-editing/modify-tasks.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
+---
+title: Add, Edit and Delete Tasks
+author: Anna Kondratova
+---
+# Add, Edit and Delete Tasks
+
+## Add a Task
+
+Right-click a task and select the "Add" [context menu item](~/interface-elements-for-web/articles/gantt/context-menu.md) or click the "Add Task" [toolbar item](~/interface-elements-for-web/articles/gantt/toolbar.md) to create a task:
```
Comment (albertov05) [inline interface-elements-for-web/articles/gantt/data-editing/modify-tasks.md:9]:
```suggestion
To create a task, right-click an existing task and select the "Add" [context menu item](~/interface-elements-for-web/articles/gantt/context-menu.md), or click the "Add Task" [toolbar item](~/interface-elements-for-web/articles/gantt/toolbar.md):
```

File: interface-elements-for-web/articles/gantt/data-editing/modify-tasks.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+
+- "New Task" - Appends a new task at the same level as the selected task.
```
Comment (albertov05) [inline interface-elements-for-web/articles/gantt/data-editing/modify-tasks.md:11]:
```suggestion
- "New Task" - Adds a new task at the same level as the selected task.
```

File: interface-elements-for-web/articles/gantt/data-editing/modify-tasks.md [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+
+## Edit a Task
+
+Use one of the approaches described below to edit a task.
+
+* Double-click a task in the task list. 
+
+    ![](~/interface-elements-for-web/images/Gantt/inline-editing.png)
+
+*  Right-click a task and select the "Task Details" context menu item (or double-click a task in the chart area).
```
Comment (albertov05) [inline interface-elements-for-web/articles/gantt/data-editing/modify-tasks.md:27]:
```suggestion
* Right-click a task and select the "Task Details" context menu item (or double-click a task in the chart area).
```

File: interface-elements-for-web/articles/gantt/data-editing/modify-tasks.md [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+
+    ![](~/interface-elements-for-web/images/Gantt/popup-edit-form.png)
+
+* Hover the mouse pointer over the task you want to modify.
```
Comment (albertov05) [inline interface-elements-for-web/articles/gantt/data-editing/modify-tasks.md:31]:
```suggestion
* Move the mouse pointer over the task you want to modify.
```

File: interface-elements-for-web/articles/gantt/data-editing/modify-tasks.md [hunk 11]
```diff
@@ -0,0 +1,67 @@
...
+
+    ![](~/interface-elements-for-web/images/Gantt/drag-and-drop.png)
+
+    **Move** - Hold on the task and drag it to change its start and end time.
```
Comment (albertov05) [inline interface-elements-for-web/articles/gantt/data-editing/modify-tasks.md:35]: I'm not sure what you mean by "Hold on the task". Do you mean:

"Drag the edge of a task to change its start and end time."

If not, please clarify.

File: interface-elements-for-web/articles/gantt/data-editing/modify-tasks.md [hunk 12]
```diff
@@ -0,0 +1,67 @@
...
+
+    ![](~/interface-elements-for-web/images/Gantt/moving.png)
+
+    **Resize** - Drag the resize handles at the task's edges to change the task duration.
```
Comment (albertov05) [inline interface-elements-for-web/articles/gantt/data-editing/modify-tasks.md:39]:
```suggestion
    **Resize** - Drag a resize handle (located at a task's edge) to change the task duration.
```

File: interface-elements-for-web/articles/gantt/data-editing/modify-tasks.md [hunk 13]
```diff
@@ -0,0 +1,67 @@
...
+
+    ![](~/interface-elements-for-web/images/Gantt/resizing.png)
+
+    **Change progress** - Move the slider to change the task's progress.
+
+    ![](~/interface-elements-for-web/images/Gantt/change-progress.png)
+
+## Save Changes
+
+The Gantt saves the changes right after you have edited a task in the task list or used drag and drop functionality.
```
Comment (albertov05) [inline interface-elements-for-web/articles/gantt/data-editing/modify-tasks.md:49]:
```suggestion
The Gantt saves changes right after you edit a task in the task list or perform a drag-and-drop operation.
```

File: interface-elements-for-web/articles/gantt/data-editing/modify-tasks.md [hunk 14]
```diff
@@ -0,0 +1,67 @@
...
+
+![](~/interface-elements-for-web/images/Gantt/manage-tasks.gif)
+
+In the "Task Details" dialog, click OK to save the changes.
```
Comment (albertov05) [inline interface-elements-for-web/articles/gantt/data-editing/modify-tasks.md:53]:
```suggestion
In the "Task Details" dialog, click OK to save changes. 
```

File: interface-elements-for-web/articles/gantt/data-editing/modify-tasks.md [hunk 15]
```diff
@@ -0,0 +1,67 @@
...
+
+![](~/interface-elements-for-web/images/Gantt/manage-tasks.gif)
+
+In the "Task Details" dialog, click OK to save the changes. 
+
+![](~/interface-elements-for-web/images/Gantt/dialog-ok.png)
+
+## Remove a Task
+
+Right-click a task and select the 'Delete Task' [context menu](~/interface-elements-for-web/articles/gantt/context-menu.md) or [toolbar](~/interface-elements-for-web/articles/gantt/toolbar.md) item to remove a task.
```
Comment (albertov05) [inline interface-elements-for-web/articles/gantt/data-editing/modify-tasks.md:59]:
```suggestion
To remove a task, right-click a task and select the 'Delete Task' [context menu](~/interface-elements-for-web/articles/gantt/context-menu.md) item or [toolbar](~/interface-elements-for-web/articles/gantt/toolbar.md) item.
```

File: interface-elements-for-web/articles/gantt/data-editing/modify-tasks.md [hunk 16]
```diff
@@ -0,0 +1,67 @@
...
+
+![](~/interface-elements-for-web/images/Gantt/delete-task.png)
+
+## Undo/Redo Changes
+
+Use Ctrl+Z and Ctrl+Y or "Undo" and "Redo" [toolbar items](~/interface-elements-for-web/articles/gantt/toolbar.md) to roll back changes.
```
Comment (albertov05) [inline interface-elements-for-web/articles/gantt/data-editing/modify-tasks.md:65]:
```suggestion
Press Ctrl+Z or Ctrl+Y (or click "Undo" or "Redo" in the [toolbar](~/interface-elements-for-web/articles/gantt/toolbar.md)) to cancel or reverse changes.
```

File: interface-elements-for-web/articles/gantt/toolbar.md [hunk 17]
```diff
@@ -0,0 +1,22 @@
...
+# Toolbar
+
+![Gantt - toolbar](~/interface-elements-for-web/images/Gantt/toolbar.png)
+
+The toolbar provides the following commands:
+
+| Command | Description |
+|---|---|
+| Undo | Reverses the last action. |
+| Redo | Repeats the undone action. |
```
Comment (albertov05) [inline interface-elements-for-web/articles/gantt/toolbar.md:15]:
```suggestion
| Redo | Restores the previously undone action. |
```

File: interface-elements-for-web/articles/gantt/toolbar.md [hunk 18]
```diff
@@ -0,0 +1,22 @@
...
+| Add Task | Adds a task. |
+| Delete Task | Deletes a task. |
+| Collapse Tasks  | Collapses all tasks. |
+| Expand Tasks | Expands all tasks. |
+| Zoom In | Zooms in the chart. |
```
Comment (albertov05) [inline interface-elements-for-web/articles/gantt/toolbar.md:20]:
```suggestion
| Zoom In | Zooms in on the chart. |
```

File: interface-elements-for-web/articles/gantt/toolbar.md [hunk 19]
```diff
@@ -0,0 +1,22 @@
...
+| Zoom In and Zoom Out | Zooms out the chart. |
```
Comment (albertov05) [inline interface-elements-for-web/articles/gantt/toolbar.md:21]:
```suggestion
| Zoom In and Zoom Out | Zooms out of the chart. |
```

File: interface-elements-for-web/articles/grid.md [hunk 20]
```diff
@@ -4,7 +4,7 @@ author: Svetlana Nikulina
 legacyId: 4854
 ---
 # Grid
-This section describes the Grid control's capabilities. It represents data in a tabular form, supports data editing, sorting, grouping, filtering, summary calculation, column resizing, data exporting and using data cell bands. 
+This section describes the Grid control's capabilities. It represents data in a tabular form, supports data editing, sorting, grouping, filtering, summary calculation, column resizing, data exporting and using data cell bands.
```
Comment (albertov05) [inline interface-elements-for-web/articles/grid.md:7]:
```suggestion
This section describes Grid control capabilities. A grid represents data in tabular form, and supports data editing, sorting, grouping, filtering, summary calculation, column resizing, data export and data cell bands.
```

File: interface-elements-for-web/articles/gantt/data-editing/modify-dependencies.md (hunk 2 above)
Comment (AnnaKondratova) [reply interface-elements-for-web/articles/gantt/data-editing/modify-dependencies.md:11]: Yes, you are right. Applied.

File: interface-elements-for-web/articles/gantt/data-editing/modify-tasks.md (hunk 11 above)
Comment (AnnaKondratova) [reply interface-elements-for-web/articles/gantt/data-editing/modify-tasks.md:35]: What you describe is resizing - change start **or** end time.

Moving is changing both start **and** end times.

Hold is to move the mouse pointer over the task and press on it.

image

File: interface-elements-for-web/articles/gantt/toolbar.md (hunk 19 above)
Comment (AnnaKondratova) [reply interface-elements-for-web/articles/gantt/toolbar.md:21]: Yes, I've missed this mistake.

Comment (AnnaKondratova) [reply interface-elements-for-web/articles/gantt/toolbar.md:21]: updated to Zoom Out

File: interface-elements-for-web/articles/gantt/data-editing/modify-tasks.md (hunk 11 above)
Comment (albertov05) [reply interface-elements-for-web/articles/gantt/data-editing/modify-tasks.md:35]: Thank you for the detailed explanation. Can we say something like:

**Move** - Drag the task's left boundary to change its start time. Drag the right boundary to change its end time.

File: interface-elements-for-web/articles/gantt/toolbar.md (hunk 19 above)
Comment (albertov05) [reply interface-elements-for-web/articles/gantt/toolbar.md]: Thank you. Also, change "Zooms out the chart." to "Zooms out of the chart."

File: interface-elements-for-web/articles/gantt/data-editing/modify-tasks.md (hunk 11 above)
Comment (AnnaKondratova) [reply interface-elements-for-web/articles/gantt/data-editing/modify-tasks.md:35]: The first sentence is better for me, thank you. Applied.

**Pull request merged**